src/main/java/org/truffleruby/core/regexp/MatchDataNodes.java

```diff
@@ -298,6 +342,9 @@ public DynamicObject toA(VirtualFrame frame, DynamicObject matchData) {
             return createArray(objects, objects.length);
         }
 
+        private static Object[] getCaptures(Object[] values) {
+            return ArrayUtils.extractRange(values, 1, values.length);
```

Comment:
Is this copy necessary? Could it skip the first index in the following for loop?